cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -86,16 +115,17 @@ spec:
                   fieldPath: metadata.namespace
           command:
             - /pod_nanny
+            - --config-dir=/etc/config
             - --cpu={{ base_metrics_cpu }}
             - --extra-cpu={{ metrics_cpu_per_node }}m
             - --memory={{ base_metrics_memory }}
-            - --extra-memory={{ metrics_memory_per_node }}Mi
+            - --extra-memory={{metrics_memory_per_node}}Mi
```

Comment:
Why? Either keep the spaces in all params or remove from all of them.

Reply:
Done